gradients.json

```diff
@@ -450,4 +450,8 @@
 {
     "name": "Master Card",
     "colors": ["#f46b45", "#eea849"]
+}
```

Comment:
Seems like you were missing a comma after the previous closing bracket. No worries though, I'll fix it. Thanks for contributing!